packages/yew-macro/src/html_tree/html_component.rs

```diff
@@ -120,8 +120,8 @@ impl ToTokens for HtmlComponent {
 
         tokens.extend(quote_spanned! {ty.span()=>
             {
-                #[allow(clippy::unit_arg)]
```

Comment:
components with empty arguments would trigger the unit arg warning:
Previously the now `__yew_props` was inlined in the `new` call. When there are no properties (e.g. `html_nested!{<MyComponent/>}`), `#build_props` would expand to `MyComponent::Properties::builder().build()` which is the unit type, triggering the unit arg warning.
Extracting variable solves that.